doc/design/requirements.md

```diff
@@ -0,0 +1,69 @@
+# Requirements and goals
+
+Includes the overlapping projects of TiUp, tiup-cluster, and tiup-playground.
```

Comment:
We use the word `TiUP` when treated it as a production (consistent with TiDB TiKV), and use the word `tiup` when used it in the command line context.

Reply:
Why TiUP not TiUp? I think the second is better since 'up' is one word like 'Ti', not an acronym like 'KV' or 'DB'

Reply:
I think we have n’t considered it carefully enough and use the word `TiUP` directly in all documents... Nick is right but it ’s weird to fix this because that many users already know this product and it's name.